lib/levels.js

```diff
@@ -52,10 +52,10 @@ function genLsCache (instance) {
   const serializer = instance[serializersSym][levelSerializerSym]
   const { labels } = instance.levels
   const cache = {}
-  Object.keys(labels).forEach(number => {
-    const level = serializer(labels[number], Number(number))
-    cache[number] = JSON.stringify(level).slice(0, -1)
-  })
+  for (const label in labels) {
```

Comment:
Should a `hasOwnProperty` check be used here?

Reply:
don't we control the `labels` object? If the user can (legitimately, as per API) overwrite the labels object then yes. If they can't then it's not needed

Reply:
https://github.com/pinojs/pino/blob/d237668d71e4e62224269eeef0b1b6dc4d668dbd/lib/levels.js#L119-L136

Reply:
I've added a stricter check in https://github.com/pinojs/pino/pull/775/commits/489a6cf95573c900651518e078143122aa456d05, but with or without it there are no test that will fail.
How can we test it?

Reply:
by putting enumerable properties on the prototype of the labels object

if you can't do that via the API then the check isn't necessary